src/native/src/components/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from "./Button";
 export * from "./Form";
-export * from "./Text";
-export * from "./Icon";
+export * from "./BaseText";
+export * from "./BaseIcon";
```

Comment:
Aí no caso, a gente vai usar como fora da lib qnd for importar em outro projeto?

```ts
import { BaseText } from 'design-system';
```

ou

```ts
import { Text } from 'design-system';
```

Só pra eu entender o padrão, por mim os dois são bons

Reply:
Podemos usar sempre como Base mesmo, acho q fica mais claro e fácil de não confundir